app/templates/electron/ts/typings.d.ts

```diff
@@ -0,0 +1,9 @@
+export {}
```

Comment:
Won't this work if added directly on the top of `app/templates/electron/ts/main-process/electron-main.ts`?
I think it should

Reply:
Tried adding `export {}` to the top of `app/templates/electron/ts/main-process/electron-main.ts` but that failed:
> TS2339: Property '__statics' does not exist on type 'Global'.
> Augmentations for the global scope can only be directly nested in external modules or ambient module declarations.

Reply:
Ok, I'll check this myself :)